### PR TITLE
fix #175: add action-outline type to button

### LINKED ISSUE #175: Add "action-outline" / "action-secondary" button type
### What problem does this feature solve?

In the Fipster team, we are currently replacing Vuetify components with Six Webcomponents in our applications. We commonly use the blue outline button, which is not present in the library: 
image

After discussions with our designer, Dimitris Papadopoulos, he suggested that we introduce an additional button for our use case.

### What does the proposed API look like?

- introduce new button type "action-outline" or "action-secondary"

### PR DESCRIPTION
**Pull request by franckdervaux**
### 🔗 Linked issue
#175 

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Add a new `action-outline` value for the `type` property of the button component.  
The styling for this new type has been reviewed with Dimitris Papadopoulos. 

Resolves #175 
### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] It's submitted to the `main` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] I have updated the documentation accordingly.
- [x] All tests are passing
- [ ] New/updated tests are included
- [x] I have updated the "upcoming" section inside _docs/changelog.md_ explaining the changes I contributed

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (pennal) [inline .vscode/launch.json:1]: Is this file needed? I think its a debugging thing and does not truly belong here

**Review by pennal — COMMENTED** [review]
Remove the unneeded file but otherwise it looks good to me

**Comment by franckdervaux** [comment]
Added commit fixing the doc

**Review by davidemarcoli — APPROVED** [review]

**Review by ibirrer — APPROVED** [review]
